src/actions/map.action-creator.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+import { PAN_MAP, } from './types';
+
+export function panMapActionCreator(panToLat, panToLng){
+   return {
+    type: PAN_MAP,
+    payload: {panToLat:panToLat,panToLng:panToLng}
```

Comment:
since the key and the value are the same, you can just write this as:

`payload: { panToLat, panToLng }`